src/components/Layout/Layout.tsx

```diff
@@ -106,6 +108,7 @@ export const Layout = ({
   const basePath = 'docs.amplify.aws';
   const metaUrl = url ? url : basePath + router.asPath;
   const pathname = router.pathname;
+  const shouldShowGen2Banner = GEN2BANNER_URLS.includes(router.asPath);
```

Comment:
For launch, we can set this to false and when we're ready to launch gen2 pages we can reset it back to `GEN2BANNER_URLS.includes(router.asPath);`